extensions/notification/NotificationConfigManager.js

```diff
@@ -390,11 +418,13 @@ class NotificationConfigManager {
      * @return {undefined}
      */
     init(cb) {
-        return this._listBucketsWithConfig((err, buckets) => {
-            if (err) {
-                return cb(err);
-            }
-            return this._updateLocalStore(buckets, cb);
+        return this._checkConfigurationParentNode(() => {
```

Comment:
Should we handle the case when the `_checkConfigurationParentNode` method fails? (related to the previous comment)

Reply:
Same as above.